lib/fs.js

```diff
@@ -0,0 +1,129 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const { promisify } = require('util');
+
+const copyFile = promisify(fs.copyFile);
+const symlink = promisify(fs.symlink);
+const readlink = promisify(fs.readlink);
+const lstat = promisify(fs.lstat);
+const readdir = promisify(fs.readdir);
+const mkdir = promisify(fs.mkdir);
+const rmdir = promisify(fs.rmdir);
+const unlink = promisify(fs.unlink);
+
+const rmDir = (current, callback) => {
```

Comment:
There is a separate PR for that https://github.com/metarhia/common/pull/252

Reply:
I realised, that I can't catch moment when the last file is copied to call callback(null) at the end, so I was suggested to use promises.

Reply:
If you answer on the review summary don't post it on unrelated review comment.